tests/integration/test_asynchronous_metrics_pk_bytes_fields/test.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import time
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance(
+    "node",
+    main_configs=["configs/asynchronous_metrics_update_period_s.xml"],
+)
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+
+    finally:
+        cluster.shutdown()
+
+
+def greater(a, b):
+    return b > a
+
+
+def lesser(a, b):
+    return b < a
+
+
+def query_until_condition(a, b, condition, retries=20, timeout=60, delay=0.5):
```

Comment:
check `assert_eq_with_retry` from `tests/integration/helpers/test_tools.py`.

Reply:
Yeah, I checked it out at the time of writing the test, it asserts for equality but I was looking for greater or lesser comparisons, so went this way.